src/org/hammerhead226/sharkmacro/actions/ActionListParser.java

```diff
@@ -87,9 +72,32 @@ public ActionList toObject() {
 		}
 
 		ActionList al = new ActionList(list);
-		putInCache(al);
 
 		return al;
 	}
 
+	/**
+	 * This method generates a new filename to be used for saving a new file. For
+	 * example, if the newest file in the storage directory is
+	 * {@code prefix0003.csv}, the method will return {@code prefix0004.csv}.
+	 * 
+	 * @return a new complete filename in the prefix + number naming convention
+	 */
+	public static String getNewFilename() {
+		return Constants.ACTIONLIST_DEFAULT_PREFIX + String.format("%04d",
+				findLatestNumberedFile(Constants.ACTIONLIST_STORAGE_DIRECTORY, Constants.ACTIONLIST_DEFAULT_PREFIX) + 1)
+				+ ".csv";
+	}
+
+	/**
+	 * This method finds the newest file named with prefix + number naming
+	 * convention in the storage directory.
+	 * 
+	 * @return the complete filename of the latest (highest numbered) file in the
+	 *         storage directory
+	 */
+	public static String getNewestFilename() {
+		return Constants.ACTIONLIST_DEFAULT_PREFIX + String.format("%04d",
+				findLatestNumberedFile(Constants.ACTIONLIST_STORAGE_DIRECTORY, Constants.ACTIONLIST_DEFAULT_PREFIX));
+	}
```

Comment:
Shouldn't this call `getNewestFilename(Constants.ACTIONLIST_STORAGE_DIRECTORY, Constants.ACTIONLIST_DEFAULT_PREFIX)`?